bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "customapps-medallia-library-FileSaver.js",
+  "version": "0.0.1",
```

Comment:
I prefer 1.1.20151003-medallia-1 to easy track in which point we start the fork and from which version is out code.